lib/constants/constants.dart

```diff
@@ -1,34 +1,52 @@
+import 'dart:ui';
+
 class Constants {
```

Comment:
You can separated this file into diferrent kind of const files, one for constants like (Color, Strings), one for dimens (All the values that are doubles like posterDetailYAlignment, ...)